src/udev/fuzz-udev-rules.c

```diff
@@ -27,7 +27,7 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
         fflush(f);
 
         assert_se(rules = udev_rules_new(RESOLVE_NAME_EARLY));
-        r = udev_rules_parse_file(rules, filename, NULL);
+        r = udev_rules_parse_file(rules, filename, false, NULL);
```

Comment:
The short comment should be added in the commit "udev-rules: move udev_check_rule_line() invocation".